pkg/entities/nfts.go

```diff
@@ -111,11 +111,6 @@ func (e *Entity) GetNFTDetail(symbol, tokenID, guildID string, queryByAddress bo
 	collection := collections[0]
 	data, err := e.getTokenDetailFromIndexer(collection.Address, tokenID)
 	if err != nil {
-		if strings.Contains(err.Error(), "not found") {
-			return &response.IndexerGetNFTTokenDetailResponseWithSuggestions{
-				Data: nil,
-			}, nil
-		}
```

Comment:
can we standardize all not found errors to one `errorA`? then check with if errors.Is(err, errorA) .............
pls try checking in `pkg/model/errors/base.go` and ``pkg/model/errors/error.go`